Okta.Xamarin/Okta.Xamarin.Android/MainActivity.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+using Android.App;
+using Android.Content.PM;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using Android.OS;
+using Android.Content;
+using Xamarin.Forms;
+
+namespace Okta.Xamarin.Droid
```

Comment:
Is this the expected namespace? Should it be `Okta.Xamarin.Android` instead?

Reply:
This namespace is part of the VS Xamarin project template and is used extensively by generated code; attempting to change it has caused a number of issues.